tests/parsers/test_utils.py

```diff
@@ -31,6 +31,7 @@ def assert_allotrope_dicts_equal(expected: DictType, actual: DictType) -> None:
         r"\['measurement identifier'\]",
         r"\['data source identifier'\]",
         r"\['calculated data identifier'\]",
+        r"\['analyte identifier'\]",
```

Comment:
I really don't like the idea of excluding this regex pattern from all tests. So much so that I just went off and made #259.

Could you please make this PR depend on #259, then pass in
```
identifiers_to_exclude = ["analyte identifier", "measurement identifier"]
```

when you call `assert_allotrope_dicts_equal` in `tests/parsers/luminex_xponent/luminex_xponent_parser_test.py`
? 

This way, we don't exclude more regex paths everywhere than we need to.